### DISCUSSION TITLE
Better way to signup using api

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#4031] [Q&A] [answered]
is there any other way to get token for newly signed up user, or that first creating a record and then calling auth-with-password (because it will cause 2 api requests)

**Comment by ganigeorgiev** [comment, marked as answer]
No, at least not out of the box.

Also note that with v0.20+ there is now option to disallow authentication for unverified users, so if you want to perform it in 1 request this will not work when submitting the request as regular user because users will not have a verified email.
